package.json

```diff
@@ -18,8 +18,8 @@
     "browser-test-remote-ie": "env-cmd .env.local nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "env-cmd .env.local nightwatch --config nightwatch.js --env safari",
     "browser-test-remote-parallel": "env-cmd .env.local nightwatch --config nightwatch.js --env safari,ie,default,chrome",
-    "_http-serve": "execr --silent http-server .",
-    "_selenium": "execr --silent selenium-standalone start",
+    "_serve": "execr --silent http-server . # means the task will be silent - no logging to the terminal",
+    "_selenium": "execr --silent selenium-standalone start # means the task will be silent - no logging to the terminal",
```

Comment:
I like the clever approach of including a comment in JSON as a comment in shell, however I'm not sure it is a safe approach. Does it work on Windows?